src/scripts/Update.ts

```diff
@@ -1420,6 +1420,207 @@ class Update implements Saveable {
+                [562.1, 562.01],
+                [618.1, 618.01],
+                [646.1, 646.01],
+                [646.2, 646.02],
+                [647.1, 647.01],
+                [648.1, 648.01],
+                [681.1, 681.01],
+                [710.1, 710.01],
+                [710.2, 710.02],
+                [710.3, 710.03],
+                [711.1, 711.01],
+                [711.2, 711.02],
+                [711.3, 711.03],
+                [791.1, 791.01],
+                [792.1, 792.01],
+                [801.1, 801.01],
+                [845.1, 845.01],
+                [845.2, 845.02],
+                [849.1, 849.01],
+                [869.01, 869.06],
+                [869.02, 869.07],
+                [869.03, 869.04],
+                [869.04, 869.01],
+                [869.05, 869.02],
+                [869.06, 869.08],
+                [869.07, 869.03],
+                [869.08, 869.05],
+                [869.11, 869.16],
+                [869.12, 869.17],
+                [869.13, 869.14],
+                [869.14, 869.11],
+                [869.15, 869.12],
+                [869.16, 869.18],
+                [869.17, 869.13],
+                [869.18, 869.15],
+                [869.21, 869.26],
+                [869.22, 869.27],
+                [869.23, 869.24],
+                [869.24, 869.21],
+                [869.25, 869.22],
+                [869.26, 869.28],
+                [869.27, 869.23],
+                [869.28, 869.25],
+                [869.31, 869.36],
+                [869.32, 869.37],
+                [869.33, 869.34],
+                [869.34, 869.31],
+                [869.35, 869.32],
+                [869.36, 869.38],
+                [869.37, 869.33],
+                [869.38, 869.35],
+                [869.41, 869.46],
+                [869.42, 869.47],
+                [869.43, 869.44],
+                [869.44, 869.41],
+                [869.45, 869.42],
+                [869.46, 869.48],
+                [869.47, 869.43],
+                [869.48, 869.45],
+                [869.51, 869.56],
+                [869.52, 869.57],
+                [869.53, 869.54],
+                [869.54, 869.51],
+                [869.55, 869.52],
+                [869.56, 869.58],
+                [869.57, 869.53],
+                [869.58, 869.55],
+                [869.61, 869.66],
+                [869.62, 869.67],
+                [869.63, 869.64],
+                [869.64, 869.61],
+                [869.65, 869.62],
+                [869.66, 869.68],
+                [869.67, 869.63],
+                [869.68, 869.65],
+                [875.1, 875.01],
+                [876.1, 876.01],
+                [877.1, 877.01],
+                [888.1, 888.01],
+                [889.1, 889.01],
+                [892.1, 892.01],
+                [893.1, 893.01],
+                [898.1, 898.01],
+                [898.2, 898.02],
+            ];
+
+            consistentIDs.forEach(([oldID, newID]) => {
```

Comment:
This isn't going to work for the Alcremie ones, you are overwiting data of a different Alcremie, so it won't be there when you try to move that id later. Osherwott is working on a method of rotating ids around in #3340, I would wait for that before these Alcremie changes.

eg .53 overwrites into .54, and then later you try to move .54 into .51, but .54 now contains stats for the old .53